runbot/runbot.py

```diff
@@ -1225,7 +1254,7 @@ def repo(self, repo=None, search='', limit='100', refresh='', **post):
         repo_ids = repo_obj.search(cr, uid, [])
         repos = repo_obj.browse(cr, uid, repo_ids)
         if not repo and repos:
-            repo = repos[0] 
+            repo = repos[0]
```

Comment:
It's a IDE issue. You can either try to turn it off (I don't know how) or review git diff before make a commit and revert such automatic changes